lib/mix/tasks/beacon.gen.site.ex

```diff
@@ -33,6 +33,8 @@ defmodule Mix.Tasks.Beacon.Gen.Site.Docs do
     * `--host` (optional) - If provided, site will be served on that host.
     * `--port` (optional) - The port to use for http requests. Only needed when `--host` is provided.  If no port is given, one will be chosen at random.
     * `--secure-port` (optional) - The port to use for https requests. Only needed when `--host` is provided.  If no port is given, one will be chosen at random.
+    * `--endpoint` (optional) - The name of the Endpoint Module for your site. If not provided, a default will be generated based on the `site`.
```

Comment:
Depending on the name, Igniter will try to create files in a path that users may not expect. For eg:

```
--endpoint DockYardWeb.Endpoint
```

Creates the file `lib/dock_yard_web/endpoint.ex`

Which is expected because it follows the Phoenix convention. Are we just follow that pattern or find a way to overwrite the path name as well?